code/modules/mob/dead/observer/observer.dm

```diff
@@ -300,10 +301,32 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 				A.overlays += source
 				source.layer = old_layer
 				source.plane = old_plane
+		if(revive_alert_promt)
+			INVOKE_ASYNC(src, .proc/reenter_corpse_prompt)
+
 	to_chat(src, "<span class='ghostalert'><a href=?src=[UID()];reenter=1>(Click to re-enter)</a></span>")
 	if(sound)
 		src << sound(sound)
 
+/mob/dead/observer/proc/reenter_corpse_prompt()
+	if(!client)
+		return
+	var/ans = alert(src, "Re-enter your corpse?", "Someone is trying to revive you...", "Yes", "No", "Not Until Next Time")
```

Comment:
To keep consistent with other prompts, it should probably be:
```suggestion
	var/ans = alert(src, "Re-enter your corpse?", "Someone is trying to revive you...", "Yes", "No", "Not this round")
```

Reply:
But the variable for this answer option is live until ghost soul is returned into the mob. I'm not that fluent in english so I kept it simple "Not Until Next Time"

Reply:
Oh ok, I see what its suppose to do now. Still, "Not until next time" is a bit confusing on what it means. But, I don't have a good suggestion for a replacement.